katas/langtonant/richard/langtonant.test.js

```diff
@@ -0,0 +1,158 @@
+import { isSquareBlack } from './verifiers.js';
```

Comment:
Lets create one .test.js file for each module that you have, that way, this file will not end up being huge, and also, will be easier to navigate in the tests created for your modules